src/main/java/nl/dtls/fairdatapoint/entity/index/IndexSettings.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * The MIT License
+ * Copyright © 2017 DTL
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package nl.dtls.fairdatapoint.entity.index;
+
+import lombok.*;
+import org.bson.types.ObjectId;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Document
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@EqualsAndHashCode
+public class IndexSettings {
+    @Id
+    protected ObjectId id;
+
+    private List<String> denyList;
+
+    public static IndexSettings getDefault() {
+        return new IndexSettings(ObjectId.get(), new ArrayList<>());
```

Comment:
Just wondering, should the default denylist settings include localhost?

Reply:
Something like `^(http|https)://localhost(:[0-9]+){0,1}.*$`?

Reply:
Something like that, yes. I'm thinking about scenarios where this would not apply, like having a local setup to test things. In that case the user would have to manually remove the entry from the denylist. Would that be reason enough not to have it in the defaults?

Reply:
I think having the default regex there is actually a good idea 👍 It serves as a good example and with admin UI that we will manage in the same release `1.8.0`, it will be easy to adjust the denylist.

Reply:
Now, when I am about to implement it... Wouldn't be better to the default settings as a database migration? Then, we can produce an error (and log it) if there is no "IndexSettings" in database. What do you think?

Reply:
Sorry, I have missed this comment. Where the defaults are stored shouldn't really matter, so if the migration approach is more robust that's all for the better.